crates/configuration/src/configuration.rs

```diff
@@ -17,7 +17,7 @@ pub const CONFIGURATION_FILENAME: &str = "configuration.json";
 pub const CONFIGURATION_JSONSCHEMA_FILENAME: &str = "schema.json";
 
 /// The parsed connector configuration.
-#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize, JsonSchema)]
```

Comment:
The lack of a consistent order here is going to bug me for the rest of my days.

Reply:
This should probably be an auto formatting option :)